docs/narr/viewconfig.rst

```diff
@@ -448,24 +448,23 @@ configured view.
   associated view callable.
 
 ``header``
-  This value matches one or more HTTP header names or header name/value pairs.
-
-  If ``header`` is specified, it must be a string or a sequence of strings,
-  each being a header name or a ``headername:headervalue`` pair.
-
-  If ``header`` is specified without a value (a bare header name only, e.g.,
-  ``If-Modified-Since``), the view will only be invoked if the HTTP header
-  exists with any value in the request.
-
-  If ``header`` is specified, and possesses a name/value pair (e.g.,
-  ``User-Agent:Mozilla/.*``), the view will only be invoked if the HTTP header
-  exists *and* the HTTP header matches the value requested.  When the
-  ``headervalue`` contains a ``:`` (colon), it will be considered a name/value
-  pair (e.g., ``User-Agent:Mozilla/.*`` or ``Host:localhost``).  The value
-  portion should be a regular expression.
-
-  Whether or not the value represents a header name or a header name/value
-  pair, the case of the header name is not significant.
+  This param matches one or more HTTP header names or header name/value pairs.
```

Comment:
I just realized, I think we should use [RFC 2616 terminology](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2). I wish I had thought of this earlier.

Reply:
@stevepiercy Thank you for being so thorough - I'm happy if you want to continue improving things but at this point extra suggestions can be saved for a separate PR as this one has already gone through a few iterations of review and is up to the standard set by the previous code that existed. I'm going to merge this now.